### ISSUE TITLE
Back arrow on Sudoku creation screen cancels input

### ISSUE THREAD

**Issue by GoogleCodeExporter** [thaindq/opensudoku-android#6]
```
What steps will reproduce the problem?
1. Start editing a new Sudoku
2. Press back arrow 
3. Work is lost

What is the expected output? What do you see instead?
It should save my work as the default, not discard/cancel as the default.


```

Original issue reported on code.google.com by `calam...@gmail.com` on 5 May 2009 at 3:22

**Comment by GoogleCodeExporter** [comment]
Original comment by `romario...@gmail.com` on 6 May 2009 at 11:54

**Comment by GoogleCodeExporter** [comment]
```
Fixed, it will be released in 0.6.1.
```

Original comment by `romario...@gmail.com` on 6 May 2009 at 12:55
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `romario...@gmail.com` on 1 Oct 2009 at 9:04
- Changed state: **Verified**
